SeqVision/SeqVision.py

```diff
@@ -0,0 +1,143 @@
+from typing import List, Union, Callable, Dict, Tuple
+import os
+from modules.fastq_tool_module import (
+    calculate_gc_content,
+    filter_reads_by_gc,
+    filter_length_bounds,
+    filter_quality_threshold,
+)
+from modules.dna_rna_tools_module import (
+    is_rna,
+    is_dna,
+    complement,
+    reverse,
+    transcribe,
+    reverse_complement,
+    which_palindrome,
+)
+
+
+def filter_fastq(
+    input_fastq: str,
+    output_fastq: str,
+    gc_bounds: Union[Tuple[int, int], int, float] = (0, 100),
+    length_bounds: Union[Tuple[int, int], int] = (0, 2**32),
+    quality_threshold: Union[int, float] = 0,
+) -> Dict[str, Tuple[str, str]]:
```

Comment:
В этой функции происходит запись в файл, так что стоит заменить возвращаемое значение на None
```suggestion
def filter_fastq(
    input_fastq: str,
    output_fastq: str,
    gc_bounds: Union[Tuple[int, int], int, float] = (0, 100),
    length_bounds: Union[Tuple[int, int], int] = (0, 2**32),
    quality_threshold: Union[int, float] = 0,
) -> None:
```